Gems/Atom/RHI/Vulkan/Code/CMakeLists.txt

```diff
@@ -160,7 +161,8 @@ ly_add_target(
             AZ::AzCore
         PUBLIC
             3rdParty::glad_vulkan
-)
+            3rdParty::amd_vma
```

Comment:
Why is this added here instead of Atom_RHI_Vulkan.Private.Static

Reply:
Fixed too many modules having amd_vma added.